src/Tabs/Tabs.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Flex from '../Flex';
+import Tab from './Tab';
+import { ENTER_KEY, SPACEBAR } from '../constants';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const { activeTabLabel, children } = this.props;
+
+    this.state = {
+      activeTabLabel: activeTabLabel || children[0].props.label
+    };
+  }
+
+  get activeTabContent() {
+    const { activeTabLabel } = this.state;
+    const { children } = this.props;
+    const activeTab = children.find(
+      ({ props: { label } }) => label === activeTabLabel
+    );
+
+    return activeTab.props.content;
+  }
+
+  activateTab = tab => {
+    const { label } = tab.props;
+
+    this.setState({ activeTabLabel: label });
+  };
+
+  handleKeyPress = (key, tab) => {
+    if (key === ENTER_KEY || key === SPACEBAR) {
+      this.activateTab(tab);
+    }
+  };
+
+  render() {
+    const { activeTabLabel } = this.state;
+    const { Component, children, ...props } = this.props;
+
+    return (
+      <>
+        <Component {...props}>
+          {children.map(tab => {
+            const {
+              label,
+              children: tabChildren,
+              content,
+              ...tabProps
+            } = tab.props;
```

Comment:
We rely heavily on `label`, `children`, `content` to be present on a tab, yet we don't have prop-type declarations on the `Tab` component. This feels a bit like a smell to me. Perhaps `Tab` should become `StyledTab` and we extract an actual `Tab` component which has the structural components we need?

Reply:
Good call, I've tosimplifed the interface to rely on tab IDs and the children of the tab will be what shows up in the tab